l2lsp/lsp_suggestEnv.go

```diff
@@ -0,0 +1,42 @@
+// lsp_suggestEnv.go
+package l2lsp
+
+import (
+	"path/filepath"
+	"strings"
+
+	l2envpackege "github.com/HexmosTech/lama2/l2env"
+	"github.com/rs/zerolog/log"
+)
+
+func getRelevantSearchString(request JSONRPCRequest) string {
+	if request.Params.RelevantSearchString != nil {
+		return *request.Params.RelevantSearchString
+	}
+	return ""
+}
+
+func getRequestURI(request JSONRPCRequest) string {
+	if request.Params.TextDocument.Uri == nil {
+		return ""
+	}
+	uri := *request.Params.TextDocument.Uri
+	if strings.HasPrefix(uri, "file://") {
+		return uri[len("file://"):]
+	}
```

Comment:
What other uri can be expected? Maybe just worth a quick investigation so that we don't fail silently.

Reply:
Yes there were wsl, remote ssh files, added those.

Reply:
How to give support for windows? Is there any examples in repo or should I consider that in next MR?

Reply:
For now, I recommend not failing silently. Show a popup in the extension that we don't support this yet.

Reply:
Yes done